pkg/config/config.go

```diff
@@ -47,14 +47,31 @@ type Config struct {
 // Load loads the config from a file.
 // If file is not present returns default config
 func Load(configFile string) (*Config, error) {
-	if configFile == "" {
-		log.Print("config file not provided - using default settings")
-		return createDefault(), nil
+	// User explicitly specified a config file
+	if configFile != "" {
+		return ParseConfigFile(configFile)
 	}
-	return ParseConfigFile(configFile)
+
+	defaultConfigFile := ".athens.toml"
```

Comment:
nit: let's pull this into a package level `const` so that it's easier for users to spot.